src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs

```diff
@@ -0,0 +1,905 @@
+
+            return ParseOperationResult.Success;
+        }
+
+        /// <summary>
+        /// Converts the specified span representation of a time to its TimeOnly equivalent using the specified format and style.
+        /// The format of the string representation must match the specified format exactly. The method returns a value that indicates whether the conversion succeeded.
+        /// </summary>
+        /// <param name="s">A span containing the characters representing a time to convert.</param>
+        /// <param name="format">The required format of s.</param>
+        /// <param name="result">When this method returns, contains the TimeOnly value equivalent to the time contained in s, if the conversion succeeded, or MinValue if the conversion failed. The conversion fails if the s is empty string, or does not contain a time that correspond to the pattern specified in format. This parameter is passed uninitialized.</param>
+        /// <returns>true if s was converted successfully; otherwise, false.</returns>
+        public static bool TryParseExact(ReadOnlySpan<char> s, ReadOnlySpan<char> format, out TimeOnly result) => TryParseExact(s, format, null, DateTimeStyles.None, out result);
+
+        /// <summary>
+        /// Converts the specified span representation of a time to its TimeOnly equivalent using the specified format, culture-specific format information, and style.
+        /// The format of the string representation must match the specified format exactly. The method returns a value that indicates whether the conversion succeeded.
+        /// </summary>
+        /// <param name="s">A span containing the characters representing a time to convert.</param>
+        /// <param name="format">The required format of s.</param>
+        /// <param name="provider">An object that supplies culture-specific formatting information about s.</param>
+        /// <param name="style">A bitwise combination of one or more enumeration values that indicate the permitted format of s.</param>
+        /// <param name="result">When this method returns, contains the TimeOnly value equivalent to the time contained in s, if the conversion succeeded, or MinValue if the conversion failed. The conversion fails if the s is empty string, or does not contain a time that correspond to the pattern specified in format. This parameter is passed uninitialized.</param>
+        /// <returns>true if s was converted successfully; otherwise, false.</returns>
+        public static bool TryParseExact(ReadOnlySpan<char> s, ReadOnlySpan<char> format, IFormatProvider? provider, DateTimeStyles style, out TimeOnly result) =>
+                            TryParseExactInternal(s, format, provider, style, out result) == ParseOperationResult.Success;
+        private static ParseOperationResult TryParseExactInternal(ReadOnlySpan<char> s, ReadOnlySpan<char> format, IFormatProvider? provider, DateTimeStyles style, out TimeOnly result)
```

Comment:
Nit: add extra line.
```suggestion

        private static ParseOperationResult TryParseExactInternal(ReadOnlySpan<char> s, ReadOnlySpan<char> format, IFormatProvider? provider, DateTimeStyles style, out TimeOnly result)
```